docs/references/cluster-configuration.md

```diff
@@ -127,18 +122,18 @@ The label name will be `env`, and the value will be what you provide here.
 
 *Value type:* boolean
 
-By default, authentication proof is required when accessing the HTTP API for pushing or querying logs/metrics data.
-This flag can be used to disable said authentication mechanism, allowing for unauthenticated clients to write or read data.
+By default, authentication proof is required when accessing the HTTP API for pushing or querying data.
+This flag enables or disables said authentication mechanism, allowing for unauthenticated clients to write or read data.
```

Comment:
thanks

Reply:
Whereas the logical consistency between the first part of the sentence and the second part is now disrupted:

> allowing for unauthenticated clients to write or read data.

Reply:
The passive voice improvement here should not come with logical regressions : ). You can get this right!

Reply:
Whoops.  Fixing by shortening the sentence.  Alternatively, we can abandon any changes here.  It's not critical.

Reply:
Whoops.  I'll just revert the change—it's not that important.